src/Mutation/PostObjectCreate.php

```diff
@@ -212,10 +213,16 @@ public static function mutate_and_get_payload( $post_type_object, $mutation_name
              * If the post being created is being assigned to another user that's not the current user, make sure
              * the current user has permission to edit others posts for this post_type
              */
-            if ( ! empty( $input['authorId'] ) && get_current_user_id() !== $input['authorId'] && ! current_user_can( $post_type_object->cap->edit_others_posts ) ) {
-                // translators: the $post_type_object->graphql_plural_name placeholder is the name of the object being mutated
-                throw new UserError( sprintf( __( 'Sorry, you are not allowed to create %1$s as this user', 'wp-graphql' ), $post_type_object->graphql_plural_name ) );
-            }
+	        $authorID = absint($input['authorId']);
```

Comment:
This breaks everything. If the `authorId` is provided as a *Relay ID* it will be a string.